GRDB/Core/DatabasePool.swift

```diff
@@ -185,7 +185,7 @@ extension DatabasePool {
         center.addObserver(
             self,
             selector: #selector(DatabasePool.applicationDidEnterBackground(_:)),
-            name: UIApplication.didReceiveMemoryWarningNotification,
```

Comment:
Note the mismatch between this and the selector used on the previous line.  `DatabaseQueue` does something similar, but without the typo.